{"name": "test_login_noExistUser_wrongPasswd[data0]", "status": "passed", "attachments": [{"name": "log", "source": "951e7ffe-7556-4193-a51e-f37549420190-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "data", "value": "{'用户名': 'zhuhonghao', '密码': '', '预期': '请输入密码'}"}], "start": 1587375140069, "stop": 1587375140908, "uuid": "049a9b7f-84a7-49fb-bd54-ddff9012d703", "historyId": "a359b13c7c6652e039886ee080e4ec86", "testCaseId": "d35b8259a99be3473adfcaff1df8682d", "fullName": "TestCases.test_00_login.TestLogin#test_login_noExistUser_wrongPasswd", "labels": [{"name": "parentSuite", "value": "TestCases"}, {"name": "suite", "value": "test_00_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-O2A05I4"}, {"name": "thread", "value": "17492-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "TestCases.test_00_login"}]}